### PR TITLE
Improve readability and add character count feature

### PR DESCRIPTION
**Pull request by iXerol**
1. use monospace font in text area.
2. automatically count non-whitespace characters of current file.
3. use `NSFileReadNoSuchFileError` instead of magic number 260.
4. filter swift file in file list.

### CODE REVIEW

**Comment by iXerol** [comment]
@toshi0383  There are other file reading error types, not only `NSFileReadNoSuchFileError` (260), do you need to handle all of them?
``` C
    NSFileReadUnknownError = 256,                                           // Read error (reason unknown)
    NSFileReadNoPermissionError = 257,                                      // Read error (permission problem)
    NSFileReadInvalidFileNameError = 258,                                   // Read error (invalid file name)
    NSFileReadCorruptFileError = 259,                                       // Read error (file corrupt, bad format, etc)
    NSFileReadNoSuchFileError = 260,                                        // Read error (no such file)
    NSFileReadInapplicableStringEncodingError = 261,                        // Read error (string encoding not applicable) also NSStringEncodingErrorKey
    NSFileReadUnsupportedSchemeError = 262,                                 // Read error (unsupported URL scheme)
    NSFileReadTooLargeError API_AVAILABLE(macos(10.5), ios(2.0), watchos(2.0), tvos(9.0)) = 263,             // Read error (file too large)
    NSFileReadUnknownStringEncodingError API_AVAILABLE(macos(10.5), ios(2.0), watchos(2.0), tvos(9.0)) = 264, // Read error (string encoding of file contents could not be determined)
```

**Review by toshi0383 — APPROVED** [review]
Great improvement. Thank you.

File: SwiftCodeBattleHelper/ContentView.swift [hunk 1]
```diff
@@ -111,8 +112,11 @@ struct ContentView: View {
                         Text("stdin")
                         pasteButton
                         deleteButton
+                        Spacer()
+                        countText
```
Comment (toshi0383) [inline SwiftCodeBattleHelper/ContentView.swift]: 🎨 `countText` should be positioned at the bottom-right of the code pane.
Would you prefer to take care of it, or would you like me to handle it from here?

Comment (iXerol) [reply SwiftCodeBattleHelper/ContentView.swift]: Adjusted the position, but I'm not sure if it matches your preferred style.
I don't have strong design preferences for this, feel free to tweak the UI.

**Review by toshi0383 — APPROVED** [review]
LGTM

**Pull request merged**